agents/insurance-solution/README.md

```diff
@@ -0,0 +1,74 @@
+# Task Automation Using Agents and Knowledge Base for Amazon Bedrock - Insurance Agent. 
+---
+
+## Content
+- [Overview](#overview)
+- [Agents and Knowledge Base Architecture](#agents-and-knowledge-base-architecture)
+- [Deployment Guide](#deployment-guide)
+- [Testing and Validation](#testing-and-validation)
+- [Clean Up](#clean-up)
+
+## Overview
+
+You can now use [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html) and [Knowledge base for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/knowledge-base.html) to configure specialized agents that seamlessly execute actions based on user input and your organization's data. These managed agents play conductor, orchestrating interactions between foundation models, API integrations, user conversations, and knowledge bases loaded with your data. Agents and knowledge bases allow you to **build on existing enterprise resources** to enhance user experience and automate repetitive tasks.
+
+This insurance agent sample solution combines an Agent and Knowledge base for Amazon Bedrock to automate insurance claim lifecycle actions like creating new claims, sending pending document reminders for open claims, gathering claims evidence, and searching for information on existing claims.
+
+### Demo Recording
+[<img src="design/thumbnail.png" width="100%">](https://www.youtube.com/watch?v=bv4XV7epeik "Task Automation Using Agents for Amazon Bedrock - YouTube")
```

Comment:
This is not an official AWS youtube channel. Could we work with marketing to get the video added to an official AWS channel?

Reply:
The video uses a kendra index. Is it possible to focus on the KB integration for the agent?

Reply:
Demo recording has been removed from repo while we await publishing of the correct demo to the official AWS YouTube channel.